shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -51,17 +51,7 @@ - (AppDelegate *)sfsdk_swizzled_init
     // Need to use SalesforceHybridSDKManager in hybrid apps
     [SalesforceSDKManager setInstanceClass:[SalesforceHybridSDKManager class]];
     
-    //Uncomment the following line inorder to enable/force the use of advanced authentication flow.
-    //[SFUserAcountManager sharedInstance].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationRequire;
-    // OR
-    // To  retrieve advanced auth configuration from the org, to determine whether to initiate advanced authentication.
-    //[SFUserAcountManager sharedInstance].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationAllow;
-    
-    // NOTE: If advanced authentication is configured or forced,  it will launch Safari to handle authentication
```

Comment:
We should probably still leave this note in here to ensure app developers don't forget to implement `openURL`.